x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/compliance_dashboard.tsx

```diff
@@ -308,32 +318,46 @@ export const ComplianceDashboard = () => {
       selectedTab,
       hasFindingsKspm,
       hasFindingsCspm,
+      isKspmInstalled,
+      isCspmInstalled,
     ]
   );
 
   return (
-    <CloudPosturePage query={selectedTab === 'cspm' ? getCspmDashboardData : getKspmDashboardData}>
-      <EuiPageHeader
-        bottomBorder
-        pageTitle={
-          <CloudPosturePageTitle
-            title={i18n.translate('xpack.csp.dashboard.cspPageTemplate.pageTitle', {
-              defaultMessage: 'Cloud Security Posture',
-            })}
-          />
-        }
-        tabs={tabs.map(({ content, ...rest }) => rest)}
-      />
-      <EuiSpacer />
-      <div
-        data-test-subj={DASHBOARD_CONTAINER}
-        css={css`
-          margin-left: auto;
-          margin-right: auto;
-          height: 100%;
-        `}
-      >
-        {tabs.find((t) => t.isSelected)?.content}
+    <CloudPosturePage
+      query={
+        selectedTab === 'cspm'
+          ? getCspmDashboardData.data
+            ? getCspmDashboardData
+            : undefined
+          : getKspmDashboardData.data
+          ? getKspmDashboardData
+          : undefined
+      }
+    >
```

Comment:
same goes for here. this property supposes to handle error and loading states. and since it worked fine in the previous version we need to understand what caused it to change.

I would prefer if this pr would fix the problems in `CloudPosturePage` and its `query` prop rather than changing its intended behaviour